lib/include/tick/array/alloc.h

```diff
@@ -5,7 +5,7 @@
 
 /** @file */
 
-#include "debug.h"
+#include "tick/base/debug.h"
```

Comment:
is full path necessary or a good practice or a find/replace mistake?

Reply:
Ok now full path must be used everywhere, the only exception is when two .h files are in the same folder.